Pester.psm1

```diff
@@ -227,7 +232,31 @@ function Add-AssertionOperator
 
     Add-AssertionDynamicParameterSet -AssertionEntry $entry
 }
+function Test-AssertionOperatorIsDuplicate
+{
+    param (
+        [object] $Operator
+    )
+
+    $existing = $script:AssertionOperators[$Operator.Name]
+
+    # Test SupportsArrayObject and scriptblock (as string) are identical
+    $isDuplicate = $existing -and
+        $Operator.SupportsArrayInput -eq $existing.SupportsArrayInput -and
+        $Operator.Test.ToString() -eq $existing.Test.ToString()
 
+    # Test Aliases
+    if ($Operator.Alias) {
+        foreach ($string in $Operator.Alias | Where { -not (Test-NullOrWhiteSpace $_)})
+        {
+            if ($existing.Alias -and (-not $existing.Alias -contains $string)) {
```

Comment:
Running the tests locally in PSv2, it seems to be throwing an error on this line. I've never had to write PSv2-compatible code before so I'm not sure what's wrong with this.